packages/sources/bx-digital/src/config/index.ts

```diff
@@ -0,0 +1,15 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig({
+  API_KEY: {
+    description: 'An API key for Data Provider',
+    type: 'string',
+    required: true,
+    sensitive: true,
+  },
+  API_ENDPOINT: {
+    description: 'An API endpoint for Data Provider',
+    type: 'string',
+    default: 'https://dev-cdf-stage-k8s.bxdigital.ch/securities',
```

Comment:
Do they have prod endpoint or we will be using stage for prod data ?

Reply:
We will be using this stage endpoint for testnet feed. Once we signed contract for mainnet we will be asking them for prod endpoint

Reply:
We will be using this endpoint for now